lib/modules/dosomething/dosomething_signup/dosomething_signup.module

```diff
@@ -1026,3 +1039,29 @@ function dosomething_signup_get_target_nid_for_source($source_string) {
   }
   return NULL;
 }
+
+/**
+ * Build the Guzzle HTTP Client to make requests to Gladiator.
+ *
+ * https://github.com/DoSomething/gladiator
+ * @TODO - use correct URL
+ */
+function _dosomething_signup_build_http_client() {
+  $base_url = 'http://jsonplaceholder.typicode.com';
+
+  if (libraries_load('guzzle') == TRUE) {
+    $client = new GuzzleHttp\Client(array(
+      'base_uri' => 'http://jsonplaceholder.typicode.com',
+      'defaults' => array(
+        'headers' => array(
+          'Content-Type' => 'application/json',
+          'Accept' => 'application/json'
+          ),
+        ),
+    ));
+
+    return $client;
+  }
+
+  return;
```

Comment:
maybe return `null` or `false` here, so when you call the function and then check if it work in **dosomething_signup.signup_data_form.inc** you have something to go off of?